content/en/examples/controllers/frontend.yaml

```diff
@@ -17,5 +17,5 @@ spec:
         tier: frontend
     spec:
       containers:
-      - name: php-redis
-        image: gcr.io/google_samples/gb-frontend:v3
```

Comment:
Check PR https://github.com/kubernetes/website/pull/44290
It addressed a similar issue on another page.
The image used is us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5.

Using this image will save you the changes needed in the rest of the file due to the use of nginx.

Reply:
Nice work @adityasamant25, you're correct!